app/User.php

```diff
@@ -7,15 +7,15 @@
 
 class User extends Authenticatable
 {
-    use Notifiable;
+    use Notifiable, SoftDeletes;
 
     /**
      * The attributes that are mass assignable.
      *
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'username', 'first_name', 'last_name', 'role', 'password',
```

Comment:
Format them vertically on the next line for readability.
Ex. username,
      first_name,